functions.sh

```diff
@@ -231,6 +231,7 @@ function make_all() {
 	local platform
 	local "${@}"
 	# ensure link.exe is the one from msvc
+	mv /usr/bin/link1 /usr/bin/link
```

Comment:
This should error out here since link1 is moved back at the end of the script and we do a `set -e` at the beginning.

Reply:
This was done because it was errorring out after this point so I had to move it back by hand to get the script to run subsequent times. Seemed like a good idea to leave in. Let me double check but it never caused an exit when link was already in the right place

Reply:
Ok I was wrong. I'll remove it. Should we take out these mv link altogether?